scripts/decode-resp/decode_test.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/pyroscope-io/pyroscope/pkg/storage/tree"
+)
+
+func TestDecode(t *testing.T) {
+	names := strings.Split("total,a,b,c", ",")
+	levels := [][]int{
+		{0, 3, 0, 0},             // total
+		{0, 3, 0, 1},             // a
+		{0, 1, 1, 3, 2, 0, 2, 2}, // b, c
+	}
+	in := &Input{
+		Flamebearer: &tree.Flamebearer{
+			Names:  names,
+			Levels: levels,
+		},
+	}
+	out := decodeLevels(in)
+	outJson := marshal(out)
+	expected := `{
+  "flamebearer": {
+    "levels": [
+      [
+        {
+          "name": "total",
+          "total": 3,
+          "self": 0
+        }
+      ],
+      [
+        {
+          "name": "a",
+          "total": 3,
+          "self": 0
+        }
+      ],
+      [
+        {
+          "name": "c",
+          "total": 1,
+          "self": 1
+        },
+        {
+          "name": "b",
+          "total": 2,
+          "self": 2
+        }
+      ]
+    ]
+  }
+}`
+	assert.Equal(t, expected, outJson)
```

Comment:
Please use ginkgo + gomega for testing stuff, here's an example: https://github.com/pyroscope-io/pyroscope/blob/main/pkg/storage/storage_test.go#L228

Reply:
I see. Will change that.

Reply:
I updated the test with ginkgo + gomega.